app/vmalert/manager.go

```diff
@@ -97,19 +98,33 @@ func (m *manager) update(ctx context.Context, path []string, validateTpl, valida
 	for _, og := range m.groups {
 		ng, ok := groupsRegistry[og.ID()]
 		if !ok {
+			startAt := time.Now()
 			// old group is not present in new list
 			// and must be stopped and deleted
 			og.close()
 			delete(m.groups, og.ID())
+			logger.Infof("group %q delete success, took %v seconds", og.Name, time.Since(startAt).Seconds())
```

Comment:
pls rephrase to "group %q stopped, took %v"
No need to print exactly seconds here and in next log message. Consider the following difference:
`group "TestGroup" delete success, took 8.0926e-05 (105.175µs) seconds` for `logger.Infof("group %q delete success, took %v (%v) seconds", og.Name, time.Since(startAt).Seconds(),  time.Since(startAt))`